packages/falso/src/lib/utils/isString.ts

```diff
@@ -0,0 +1,3 @@
+export const isString = (value: any) => {
+  return typeof value === 'string';
+};
```

Comment:
Please rename the file to `utils/validators.utils`.
```suggestion
export function isString(value: any): value is string {
  return typeof value === 'string';
}
```